vault/dendron.release.2021-11-09.md

```diff
@@ -0,0 +1,76 @@
+---
+id: EjPEcAkZIDgzHaVMk4teA
+title: Release Notes (version 0.68)
+desc: ''
+updated: 1636480628276
+created: 1636478484110
+---
+
+Dendron 0.68 has sprouted  🌱
+
+Take a look at the latest [Dendron wiki](https://wiki.dendron.so), and you'll see some new features directly from these highlights:
+
+* Local TOC in the right-hand nav for each page in the published docs
+* Newly combined search bar that can go directly to a note, or search note contents using the `?` prefix
+
+Also: an enhancement in this release delivers breaking changes announced last week, due to some renaming of commands, which can be referenced in the changelog.
```

Comment:
can we include a `Breaking` section underneath `Everything Else` with line items on whats breaking? The changelog is there for folks that want extra detail but the release notes should stand on their own (be self contained)